app/src/main/java/com/onegini/mobile/exampleapp/storage/FCMStorage.java

```diff
@@ -52,4 +55,11 @@ public int getAppVersion() {
   public void save() {
     editor.apply();
   }
+
+  public void removeDeprecatedGcmRegistrationId() {
```

Comment:
Maybe we should make this method private and call it always in the constructor or `save` method?